From ae30975118cd9ecb689464ec40384974da03effa Mon Sep 17 00:00:00 2001
From: Anton Gerasimov <anton@advancedtelematic.com>
Date: Fri, 10 Nov 2017 18:51:02 +0100
Subject: [PATCH 1/1] Move Cache-As-RAM memory from area mapped to ROM in QEMU

ROM has been made read-only in qemu recently (namely commit
208fa0e43645edd0b0d8f838857dfc79daff40a8), so this patch restores
compatibility between u-boot and qemu. It is still broken for me
unless I set CONFIG_SMP=n and disable lapic (i.e. revert patch
aaaa55751ab1e5a5cfa0962d604593a7e6f33ff6 in u-boot), but these are
separate issues

Signed-off-by: Anton Gerasimov <anton@advancedtelematic.com>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/x86/cpu/qemu/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/cpu/qemu/Kconfig b/arch/x86/cpu/qemu/Kconfig
index da378128fe..81444f3d9e 100644
--- a/arch/x86/cpu/qemu/Kconfig
+++ b/arch/x86/cpu/qemu/Kconfig
@@ -18,7 +18,7 @@ if QEMU
 
 config SYS_CAR_ADDR
 	hex
-	default 0xd0000
+	default 0x10000
 
 config SYS_CAR_SIZE
 	hex
-- 
2.15.0

